### PR TITLE
Add chapter for error handling

### PR DESCRIPTION
**Pull request by Brutus5000**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: pages/guide/v5/15.5-error-handling.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    margin-left: 10px;
+}
+
+.annotation-list .code-font {
+    font-family: "Courier New", Courier, monospace;
+    margin-left: 10px;
+}
+</style>
+
+Elide handles errors as defined in the JSONAPI specification. Where possible Elide will already create errors for you.
```
Comment (aklish) [inline pages/guide/v5/15.5-error-handling.md]: Can we include GraphQL here as well.

**Comment by aklish** [comment]
I had started documenting this feature for Elide 6:
https://github.com/yahoo/elide-doc/pull/172/files

I will merge this change for Elide 5 docs though once the small change above is made.  Let me know if you see any issues with what I documented in Elide 6.

**Comment by Brutus5000** [comment]
Where did you put it in your PR? I see a snippet for the error mapper, but can't find any mentions of the CustomErrorException or the handling of javax.validation annotations.

File: pages/guide/v5/15.5-error-handling.md (hunk 1 above)
Comment (Brutus5000) [reply pages/guide/v5/15.5-error-handling.md]: Done

**Comment by aklish** [comment]
> Where did you put it in your PR? I see a snippet for the error mapper, but can't find any mentions of the CustomErrorException or the handling of javax.validation annotations.

https://elide.io/pages/guide/v6/09-clientapis.html#custom-error-responses

**Review by aklish — APPROVED** [review]

**Pull request merged**